### ISSUE TITLE
Git flow commands now show "master: integer expression expected" in command line

### ISSUE THREAD

**Issue by kevinquillen** [petervanderdoes/gitflow-avh#286]
After upgrading to git 2.9.3 (OSX 10.11.6) via homebrew, when I do a git flow command with git flow avh 1.10.1, I get this message in the command line:

```
~/S >> release/1.18: git flow release finish 1.18
/usr/local/Cellar/git-flow-avh/1.10.1/libexec/bin/gitflow-common: line 79: [: -master: integer expression expected
```

Any ideas on what this is or how to fix it?

**Comment by petervanderdoes** [comment]
Is it with every git-flow command?

Could you run:

`DEBUG=yes git flow release finish 1.18` or a git-flow command that produce the above error and post the last 30-20 lines of the debug output?

**Comment by drucci** [comment]
```
DEBUG=yes git flow release finish -n 3.51.0
git --version
git version 2.10.0

```

```
+ '[' -n release/3.51.0 ']'
+ require_clean_working_tree
+ local result
+ git_is_clean_working_tree
+ git rev-parse --verify HEAD
+ git update-index -q --ignore-submodules --refresh
+ git diff-files --quiet --ignore-submodules
+ git diff-index --cached --quiet --ignore-submodules HEAD --
+ return 0
+ result=0
+ '[' 0 -eq 1 ']'
+ '[' 0 -eq 2 ']'
+ git_remote_branch_exists origin/release/3.51.0
+ '[' -n origin/release/3.51.0 ']'
++ git for-each-ref '--format=%(refname:short)' refs/remotes/origin/release/3.51.0
+ '[' -n '' ']'
+ '[' develop = develop ']'
+ _finish_from_develop
+ local opts merge_branch commit keepmsg remotebranchdeleted localbranchdeleted compare_refs_result merge_result
+ remotebranchdeleted=1
+ localbranchdeleted=1
+ flag fetch
+ local FLAG
+ eval 'FLAG=$FLAGS_fetch'
++ FLAG=1
+ '[' 1 -eq 0 ']'
+ git_remote_branch_exists origin/release/3.51.0
+ '[' -n origin/release/3.51.0 ']'
++ git for-each-ref '--format=%(refname:short)' refs/remotes/origin/release/3.51.0
+ '[' -n '' ']'
+ git_remote_branch_exists origin/master
+ '[' -n origin/master ']'
++ git for-each-ref '--format=%(refname:short)' refs/remotes/origin/master
+ '[' -n origin/master ']'
+ flag ff-master
+ local FLAG
+ eval 'FLAG=$FLAGS_ff-master'
++ FLAG=-master
+ '[' -master -eq 0 ']'
/usr/local/Cellar/git-flow-avh/1.10.1/libexec/bin/gitflow-common: line 79: [: -master: integer expression expected
+ require_branches_equal master origin/master
```

**Comment by petervanderdoes** [comment]
Got it, Fixed per version 1.10.2. Just released.
